### DISCUSSION TITLE
Form表单支持渐进增强

### DISCUSSION THREAD

**Discussion by lili21** [DouyinFE/semi-design#2056] [Ideas]
和原生表单行为保持一致，在JS disabled的情况下表单提交正常工作

**Comment by pointhalo** [comment]
具体想要的是 form 透传 action  props ？

**Reply by lili21** [reply]
每个form item组件都支持配置name，表达提交数据默认是formData 等等

**Reply by lili21** [reply]
比如现在select就不支持name属性
